src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseEventFieldDefinition.java

```diff
@@ -23,7 +23,6 @@
     private String label = null;
     private String hintText = null;
     private Boolean retainHiddenValue;
-    private String defaultValue;
```

Comment:
CaseFieldDefiniton is going to created from Definition store, may be we need this defaultValue method?

Reply:
Same as above. As per conversation this is not coming from the definition store and it is not used.